pages/app-layout/with-drawers.page.tsx

```diff
@@ -0,0 +1,170 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Copied this from #964 for manual testing.

Committing it now would not hurt. Also running this page at least through [our generic a11y test](https://github.com/cloudscape-design/components/blob/main/src/__a11y__/run-a11y-tests.ts) is better than no tests at all